lib/lobstersbot/pluggable_connection.rb

```diff
@@ -1,5 +1,19 @@
 module Lobstersbot
   module PluggableConnection
+    # Add class methods to Bot for triggers.
+    def self.included(mod)
```

Comment:
Could we just add this to `class Bot` in `exe/lobstersbot` instead of the complexity of injecting here?

```ruby
class Bot < Summer::Connection
  include Lobserverbot::PluggableConnection

  class << self
    def triggers
      @triggers ||= {}
    end
  end

  include Lobstersbot::Tell
  include :…
end
```

Loses a bunch of the complexity from here, which given the bot doesn't do anything but include `PluggableConnection` (and we're the only thing that drives the use of `trigger`) isn't too brittle?

Reply:
Just exposed `add_trigger` instead and kept the rest within `PluggableConnection`.